app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -367,8 +375,8 @@ public GetParticipantTask(String username, String password) { // must specify us
                     "\"query\" : {" +
                     "\"bool\" : {" +
                     "\"must\" : [" +
-                    "{ \"match\": { \"userName\": \"" + username + "\" }}," +
-                    "{ \"match\": { \"password\": \"" + password + "\" }}" +
+                    "{ \"term\": { \"userName\": \"" + username + "\" }}," +
+                    "{ \"term\": { \"password\": \"" + password + "\" }}" +
```

Comment:
Term queries demand an exact match, not just a "contains" match. This is the functionality we want for a user/pass check.